### PR TITLE
Add support for downloading photos and videos

### PR DESCRIPTION
**Pull request by Vaelatern**
Addresses #7 by adding a new mixin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,19 +20,19 @@ Just one - [Requests](http://docs.python-requests.org/en/latest/) takes care of
...
+| Records             | find, search, create, update, delete                  |
+| Photos              | find, search, get_original, get_size                  |
+| Projects            | find, search, create, update, delete                  |
+| Changesets          | find, search, create, update, close                   |
+| Choice Lists        | find, search, create, update, delete                  |
+| Classification Sets | find, search, create, update, delete                  |
+| Webhooks            | find, search, create, update, delete                  |
+| Videos              | find, search, get_original, get_size                  |
+| Memberships         | search                                                |
+| Roles               | search                                                |
```
Comment (Vaelatern) [inline README.md]: All this space is because my text editor makes it easy and I anticipate adding more methods to be more API-complete later.

**Comment by Vaelatern** [comment]
Rebased on master and support for downloads of Signatures added.

**Comment by Vaelatern** [comment]
Not sure, again, how to build automated tests.

**Comment by JasonSanford** [comment]
When there's a Pull Request, any commit in the branch reference in that PR will trigger a rebuild unless you have a `[skipci]` tag in your commit message (I think).

**Comment by Vaelatern** [comment]
I'm referring to building the tests, not triggering them. Sorry to leave that out.

**Comment by JasonSanford** [comment]
Ahh, ok. These instructions are working for me: https://github.com/fulcrumapp/fulcrum-python#testing

**Comment by JasonSanford** [comment]
Sorry for the long delay here, but I did finally merge your changes. I refactored things a bit to allow for downloading all media (no matter the size) from the same method call. These are available at v1.6.0

Thanks for your contribution!

**Pull request merged**